### DISCUSSION TITLE
Does Wallet Connect support communication between a Web wallet and a Web Dapp?

### DISCUSSION THREAD

**Discussion by markusait** [WalletConnect/walletconnect-monorepo#745] [Developer Support]
I know that Wallet Connect supports Desktop Web Dapp <-> Mobile Wallet App and Mobile Web Dapp <-> Mobile Wallet App communication, is this also the case for Web wallets like [Magic](https://magic.link/) ?

It appears that the communication for mobile linking as explained in the docs https://docs.walletconnect.com/mobile-linking could be used for this purpose. Is that correct? And if yes, is there a working example of a Web wallet implementing Wallet connect?

**Comment by chadyj** [comment]
Yes, WalletConnect supports communication between a web wallet and a web dapp.

**Comment by markusait** [comment]
Ok great, so one would use the React SDK for this purpose https://docs.walletconnect.com/2.0/quick-start/wallets/react-native ?

Is there a reference implementation or an example of a production web wallet implementing Wallet connect available?
